README.md

```diff
@@ -65,6 +65,10 @@ This cog will allow members to send a report into a channel where it can be revi
 
 This cog will allow users to prove they're not a bot by having to read rules and complete an action. They will then be given the verified role if they can complete this.
 
+### Feed
+
+This cog allows users to feed each other
```

Comment:
This is not in alphabetical order

Reply:
Also missing the ToC addition and Cog Summaries link

Reply:
> ToC addition and Cog Summaries link
What's this? Please explain?

Reply:
In README, there's two more sections, "Table of Contents" and "Cog Summaries" - Feed needs to be added to both of these lists too 😄

Reply:
Ah I see it now. Added to both